cql/the-basics-of-cql/cinchy-supported-functions/connections-functions.md

```diff
@@ -4,37 +4,79 @@
 
 The set of functions listed in this page are for use in the [Cinchy Connections Experience](https://cli.docs.cinchy.com/) when configuring data syncs.
 
+* [@COLUMN]
 * [​JSON\_ESCAPE​](connections-functions.md#json\_escape)
 * [​URL\_ESCAPE​](connections-functions.md#url\_escape)
 * [STRING\_ESCAPE()](connections-functions.md#string\_escape)
+* [@ID](connections-functions#id)
+
+## @COLUMN
+
+The `@COLUMN` function is used for real-time syncs where your source is an event (such as Cinchy Event Broker) and your target is REST/SOAP. 
```

Comment:
@COLUMN works in batch syncs as well...

Reply:
Currently it's REST/SOAP, but this could be expanded to any nested data structure in the future like Mongo/Kafka documents